src/components/OrderListGroups/index.js

```diff
@@ -1040,6 +1044,22 @@ export const OrderListGroups = (props) => {
     }
   }, [ordersGroup])
 
+  useEffect(() => {
+    if (socket?.socket && session?.auth) {
+      // eslint-disable-next-line no-unused-expressions
+      socket?.socket?.on('connect', () => {
+        loadOrders({ newFetch: true })
+      })
+    }
+
+    return () => {
+      if (socket?.socket) {
+        // eslint-disable-next-line no-unused-expressions
```

Comment:
Please don't avoid the warnings, fix with the best solution.

`socket && socket?.socket.....`

Reply:
the error says that the expression expected an asignament, but that asignament is not necessary for other calls so I just gonna delete de comments